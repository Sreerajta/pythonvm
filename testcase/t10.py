def fact(n):
    if n == 0:
    	return 1
    return n * fact(n - 1)



n=1
print "printing factorial of numbers upto 10 using a recursive function and a while loop"
while(n<=10):
 print fact(n)
 n=n+1
