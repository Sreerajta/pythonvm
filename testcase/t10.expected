printing factorial of numbers upto 10 using a recursive function and a while loop
1
2
6
24
120
720
5040
40320
362880
3628800
